src/mscorlib/src/System/Collections/Hashtable.cs

```diff
@@ -66,6 +66,7 @@ namespace System.Collections
     // 
     [DebuggerTypeProxy(typeof(System.Collections.Hashtable.HashtableDebugView))]
     [DebuggerDisplay("Count = {Count}")]
+    [Serializable]
```

Comment:
If TimeZone is the only reason for moving HashTable, we could also drop serialization on TimeZone. When I added it to the list, I didn't realize it's obsolete. We do have data showing it used in serialization, but it's rare (~0.001%).